src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -967,10 +967,9 @@ object FunctionRegistry {
     "stop" -> "Generate numbers up to, but not including this number.",
     "step" -> "Difference between each number in the sequence.")
 
-  registerCode("Call", (gt: Code[Int]) => CM.ret(gt),
-    """
-    Construct a :ref:`call` from an integer.
-    """, "gt" -> "integer")(int32Hr, callHr)
+  register("Call", { (gt: Int) => Call2.fromUnphasedDiploidIndex(gt) }, null)(int32Hr, callHr)
+  register("Call", { (alleles: IndexedSeq[Int], phased: Boolean) => CallN(alleles.toArray, phased) }, null)(arrayHr(int32Hr), boolHr, callHr)
```

Comment:
I'd remove the array constructor for now.

Reply:
This is needed for `to_expr` in `expression.py`.